toolkit/docs/building/building.md

```diff
@@ -159,11 +159,11 @@ Large parts of the build are parallelized. Enable this by setting the `-j` flag
 **NOTE: If you are building your toolchain packages from source, add `REBUILD_TOOLCHAIN=y`**
 ```bash
 # Build ALL packages FOR AMD64
-sudo make build-packages -j$(nproc) CONFIG_FILE= DOWNLOAD_SRPMS=y REBUILD_TOOLS=y PACKAGE_IGNORE_LIST="openjdk8 openjdk8_aarch64 shim-unsigned-aarch64"
+sudo make build-packages -j$(nproc) CONFIG_FILE= DOWNLOAD_SRPMS=y REBUILD_TOOLS=y
```

Comment:
Suggestion: since these commands are now identical, we could reword this section to provide only one example. This way we avoid making an initial impression that you need to do something different depending on the architecture.